client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -114,19 +79,56 @@ class EditorMediaModalDetailItem extends Component {
 		return true;
 	}
 
+	/**
+	 * This function return true if the image editor can be
```

Comment:
nit: 'returns'

Reply:
I some how inherited this, moving from a function to a method. I've corrected it though